plugins/modules/mount.py

```diff
@@ -93,7 +93,10 @@
   boot:
     description:
       - Determines if the filesystem should be mounted on boot.
-      - Only applies to Solaris systems.
+      - Only applies to Solaris and Linux systems.
+      - For Solaris systems, C(true) will set C(yes) as the value of mount at boot
+        in I(/etc/vfstab).
+      - For Linux systems, C(true) will add C(noauto) to mount options in I(/etc/fstab).
```

Comment:
Shouldn't documentations say that `false`(!) will add `noauto` on Linux?

Reply:
@darkdragon-001 
Ahhh, thank you for pointing it out!! I'll create another PR to fix it.

Reply:
Fixes #203